docs/plugins/plugins.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "./plugins.schema.json",
+  "plugins": [
+    {
+      "ghrepo": "aspect-build/plugin-fix-visibility"
```

Comment:
"ghrepo"?

Reply:
That's what we used in the docs repo in the json schema for rulesets. Agree that doesn't look so good for a public API